jobsAdmin/init.go

```diff
@@ -459,6 +459,29 @@ func GetJobSummary(jobID common.JobID) common.ListJobSummaryResponse {
 	}
 	part0PlanStatus := part0.Plan().JobStatus()
 
+	// This is added to let FE to continue fetching the Job Progress Summary
+	// in case of resume. In case of resume, the Job is already completely
+	// ordered so the progress summary should be fetched until all job parts
+	// are iterated and have been scheduled
+	js.CompleteJobOrdered = js.CompleteJobOrdered || jm.AllTransfersScheduled()
+
+	if part0PlanStatus != common.EJobStatus.Cancelled() &&
+		(js.CompleteJobOrdered) && (part0PlanStatus.IsJobDone()) {
+		/*
+		 * Plan file (hence part0PlanStatus.IsJobDone) is updated inline by jm.reportJobPartDoneHandler()
+		 * while the various transfer stats are updated via xferDone messages processed by handleStatusUpdateMessage().
+		 * Now that the job has completed, make sure we drain all queued-but-not-processed-yet xferDone messages so as to
+		 * return correct updated job status to our caller who may not call us again since we declare job completion.
+		 */
+		if jm.DrainXferDoneMessages() {
+			// Let's get JobStatus once again to get the updated value.
+			js = jm.ListJobSummary()
+			js.Timestamp = time.Now().UTC()
+			js.JobID = jm.JobID()
+			js.ErrorMsg = ""
+		}
```

Comment:
should we again set js.CompleteJobOrdered inside this since we refreshed js.
Technically it should be true, but could it be o/w?

Reply:
Yes we should set